components/sheet/ImportCSVSheet.tsx

```
@@ -35,29 +43,151 @@ type ImportCSVSheetProps = {
+
+  const validateCSVRow = (row: CSVRow, index: number): string[] => {
+    const errors: string[] = [];
+    const { amount, date, currencyCode } = row;
+
+    if (!amount || !date || !currencyCode) {
+      errors.push(
+        `Row ${index + 1}: Missing required fields (merchant, amount, date, or currency)`
+      );
+    }
+
+    if (amount && isNaN(Number(amount))) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)